iotexproject/iotex-token-metadata
# IOS-Token

## Token Information

- **Project official website**: 
- https://iotexsons.io


- **Project official social**:
- https://t.me/iotexsons
- https://twitter.com/theiotexsons?s=21

## Verification of Token

-  verify of your token smart contract on iotexscan:
    io1kz4s5gq78jvx03x9hyejlfz55rvyfrrykuyfnv

- add your token address to your website and list here:
   https://iotexscout.io/address/io1kz4s5gq78jvx03x9hyejlfz55rvyfrrykuyfnv#code
